<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="352894" started-at="2018-05-08T11:46:36Z" finished-at="2018-05-08T11:52:29Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="352894" started-at="2018-05-08T11:46:36Z" finished-at="2018-05-08T11:52:29Z">
      <class name="com.primeresponse.testcases.Accounts.Check_LoginIcon">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.primeresponse.testcases.Accounts.Check_LoginIcon@fc185]" name="setUp" is-config="true" duration-ms="8012" started-at="2018-05-08T17:16:36Z" finished-at="2018-05-08T17:16:44Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="testCreate_Account()[pri:0, instance:com.primeresponse.testcases.Accounts.Check_LoginIcon@fc185]" name="testCreate_Account" duration-ms="344512" started-at="2018-05-08T17:16:44Z" finished-at="2018-05-08T17:22:28Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Element Locator ://div[@id='account-search-results']/li Not found expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Element Locator ://div[@id='account-search-results']/li Not found expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at com.primeresponse.util.DriverHelper.clickOn(DriverHelper.java:235)
	at com.primeresponse.pagehelper.AccountsHelper.loginIconNotPresent(AccountsHelper.java:258)
	at com.primeresponse.testcases.Accounts.Check_LoginIcon.testCreate_Account(Check_LoginIcon.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testCreate_Account -->
        <test-method status="PASS" signature="endMethods()[pri:0, instance:com.primeresponse.testcases.Accounts.Check_LoginIcon@fc185]" name="endMethods" is-config="true" duration-ms="17" started-at="2018-05-08T17:22:28Z" finished-at="2018-05-08T17:22:28Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endMethods -->
        <test-method status="PASS" signature="afterMainMethod()[pri:0, instance:com.primeresponse.testcases.Accounts.Check_LoginIcon@fc185]" name="afterMainMethod" is-config="true" duration-ms="295" started-at="2018-05-08T17:22:28Z" finished-at="2018-05-08T17:22:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMainMethod -->
      </class> <!-- com.primeresponse.testcases.Accounts.Check_LoginIcon -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
